### ISSUE TITLE
Enhance some horse related search results

### ISSUE THREAD

**Issue by Hufkratzer** [openstreetmap/id-tagging-schema#413]
It seems that a lot of [Riding Arenas](https://wiki.openstreetmap.org/wiki/Tag:leisure%3Dpitch#Riding_arena)  are mapped as [Horseback Riding Facilities](https://wiki.openstreetmap.org/wiki/Tag:leisure%3Dhorse_riding). This may be because some horse related searches don't list Riding Arenas.

I don't know how the feature search exactly works. Therefore, I list here some horse related search results and explain what I think is missing or is wrong with them and how it can possibly be improved:

----
#### search result (en)
<img width="207" alt="horseback-riding-node-en" src="https://user-images.githubusercontent.com/5958184/156854045-166852a8-cd24-4a91-bdd2-d0de27c95b48.png">

#### What is wrong?
* `Riding Arena` is missing, see also [more general description](openstreetmap/iD#9105) 
#### How to fix? (assumption)
* Add alias `Horseback Riding Arena` to [presets/leisure/pitch/equestrian.json](https://github.com/openstreetmap/id-tagging-schema/blob/main/data/presets/leisure/pitch/equestrian.json) (already contains terms `horseback` and `riding` but this doesn't seem to be enough to get it listed when searching for `horseback riding`)
----
#### search result (same for en and en-GB)
<img width="194" alt="horse-ridng-area-en" src="https://user-images.githubusercontent.com/5958184/156854083-f0242e4d-7654-455d-b4cc-92e6f688de51.png">

#### What is wrong?
1. `Riding Arena` is missing, see also [more general description](openstreetmap/iD#9105) 
2. `Horse Racing Track` shouldn't be listed here; at least it shouldn't be listed before `Riding Arena` or `Horseback Riding Facility`.
#### How to fix? (assumption)
1. Add alias `Horseback Riding Arena` to [presets/leisure/pitch/equestrian.json](https://github.com/openstreetmap/id-tagging-schema/blob/main/data/presets/leisure/pitch/equestrian.json) (already contains terms `horse` and `riding` but this doesn't seem to be enough to get it listed when searching for `horse riding`). (`Horseback Riding Arena` has to be translated to `Horse Riding Arena` for en-GB.)
2. Don't know why `Horse Racing Track` is listed here, see [presets/leisure/track/horse_racing.json](https://github.com/openstreetmap/id-tagging-schema/blob/main/data/presets/leisure/track/horse_racing.json). Perhaps adding alias `Horseback Riding Arena` (see point 1) can help to move if below `Riding Arena`, I am not sure.
----
#### search result
<img width="185" alt="horse-node-en" src="https://user-images.githubusercontent.com/5958184/156854187-bc1dcdcb-d249-47fc-bf87-fe8673a37369.png">

#### What is wrong?
* `Riding Arena` is only at 9th position, should rather be listed near or before `Horseback Riding Facility`
#### How to fix? (assumption)
* Addng alias `Horseback Riding Arena` to [presets/leisure/pitch/equestrian.json](https://github.com/openstreetmap/id-tagging-schema/blob/main/data/presets/leisure/pitch/equestrian.json) may improve this, I am not sure.
----
#### search result
<img width="186" alt="horse-racing-node-en" src="https://user-images.githubusercontent.com/5958184/156854153-8e84050d-5944-4654-b5e3-370dda09b107.png">

#### What is wrong?
* `Horseback Riding Facility` shouldn't be listed here, but at least `Horce Racing Track` is listed before `Horseback Riding Facility`, so this may still be accepable.
#### How to fix? (assumption)
* Don't know, `horse racing` isn't in [presets/leisure/horse_riding.json](https://github.com/openstreetmap/id-tagging-schema/blob/main/data/presets/leisure/horse_riding.json)
----
#### search result
<img width="184" alt="horseback-area-en" src="https://user-images.githubusercontent.com/5958184/156858543-1157a9f4-3d08-41d2-9ebb-c1b30cb903dd.png">

#### What is wrong?
* `Horse Racing Track` shouldn't be listed here
* `Motocross` Racetrack shouldn't be listed here
* `Motorsport` Racetrack shouldn't be listed here
#### How to fix? (assumption)
* Don't know why these racetracks are listed here
----
#### search result
<img width="196" alt="riding-area-en" src="https://user-images.githubusercontent.com/5958184/156856486-e2ecb9f4-7d6e-4317-abe7-df4758e26eba.png">

#### What is wrong?
* Nothing :1st_place_medal: 
----
#### search result
![horse-stable](https://user-images.githubusercontent.com/5958184/159137829-880a35e1-b631-4511-a36e-bc1370942d89.png)
#### What is wrong?
* `Stable` ([presets/building/stable.json](https://github.com/openstreetmap/id-tagging-schema/blob/main/data/presets/building/stable.json)) not listed, see also [more general description](openstreetmap/iD#9105) 
#### How to fix? (assumption)
* Add alias `Horse Stable` to preset `Stable`

----
#### search result
![barnsearch](https://user-images.githubusercontent.com/5958184/160889634-17e40c84-a1a1-43bb-8ac3-03ff3659f015.jpg)
(similar for **shed**)
#### What is wrong?
* `Stable` (for horses) not listed. It seems that in the US they sometimes call a horse stable "barn" or "shed", see websites: [here](http://www.inspiredhomeideas.com/horseback-riding-ranch-horse-stables-barns-and-facilities/) and [here](https://www.horizonstructures.com/horse-barns) 
#### How to fix? (assumption)
* Add terms `horse barn` and `horse shed` to [presets/building/stable.json](https://github.com/openstreetmap/id-tagging-schema/blob/main/data/presets/building/stable.json), compare [presets/building/cowshed.json](https://github.com/openstreetmap/id-tagging-schema/blob/main/data/presets/building/cowshed.json) and [presets/building/sty.json](https://github.com/openstreetmap/id-tagging-schema/blob/main/data/presets/building/sty.json)

**Comment by Hufkratzer** [comment]
AFAIK, it is [ _Horse Riding_ in British English](https://dictionary.cambridge.org/dictionary/english/horse-riding), [ _Horseback Riding_ in US English](https://dictionary.cambridge.org/dictionary/english/horseback-riding) and [ _Horse Racing_ in both, British and US English](https://dictionary.cambridge.org/dictionary/english/horse-racing) (see also [wikipedia: horse racing](https://en.wikipedia.org/wiki/Horse_racing)). But I'm not a native English speaker either.

Please also note:
* The [preset](https://github.com/openstreetmap/id-tagging-schema/blob/main/data/presets/type/route/horse.json) for `type=route`+ `route=horse` is currently named _Horse Riding Route_. (Until [Dec 2020](https://github.com/openstreetmap/id-tagging-schema/commit/a554f56362d1af28ad2c32ea997d34587deb4dad) it was just _Ridng Route_.) Maybe it should be _Horseback Riding Route_ in US English, compare  [webpage](https://horseridingnetwork.com/top-14-locations-in-the-usa-for-horseback-riding-trails/); but I am not sure.
* In related issue https://github.com/openstreetmap/id-tagging-schema/issues/437 I suggest using the name _Horseback Riding **Center**_ instead of _Horseback Riding Facility_ because _Facility_ is too general and ambiguous (e.g. a riding arena can also be seen as a facility), and this probably contributed to many mistaggings.

In summary, if you want to use the long versions of the names, I would suggest:
| tag       |   US English          | British English |
| ---------------                                   | --------------- | --------------- |
| `leisure=pitch` + `sport=equestrian` | Horseback Riding Arena | Horse Riding Arena |
| `leisure=horse_riding` | Horseback Riding Center | Horse Riding Centre |
| `type=route`+ `route=horse` | Horseback Riding Route | Horse Riding Route |
| `leisure=track` + `sport=horse_racing` | Horse Racing Track | _unchanged_ |

Maybe using the British names as aliases in the presets can improve the search results in case someone searches just for "horse" (instead of "horseback"). I don't know.

**Comment by 1ec5** [comment]
`leisure=pitch` `sport=equestrian` is [one of these outdoor enclosures](https://www.openstreetmap.org/way/1021503880), right? I think “(horseback) riding ring” or “pen” is the more common term for those in American English, at least among laypeople. “Riding arena” [isn’t wrong](https://www.openstreetmap.org/way/683596971). However, if a user unfamiliar with equestrian sports sees “arena” and “center” options, I’m not sure they could distinguish between them. They might associate “arena” with [a large indoor facility](https://www.openstreetmap.org/way/645946201). That said, [equestrianism has its own lingo](https://horserookie.com/what-do-you-call-a-place-where-you-ride-horses/) so maybe it’s best to stick to that.

For `leisure=track` `sport=horse_racing`, “horse racing track” is perfectly understandable, though “horse racetrack” is the more common term.

**Comment by tyrasd** [comment]
Thank you for the clarifications.

> if you want to use the long versions of the names

My main intention was to use the a single "prefix" for all horse riding related presets, because  it would solve at least some of the sorting issues listed above: A partial match on a preset name will always be preferred over a match of one of the preset terms.

<hr> 

Would the following make sense?

| tag       |   preset name (US English) | preset alias(es) (US English) | additional preset terms
| ---------------                                   | --------------- | -- | --
| `leisure=pitch` + `sport=equestrian` | Horseback Riding Ring | Horseback Riding Arena | [round] pen, [horse] riding ring/arena
| `leisure=horse_riding` | Horseback Riding Center | Equestrian Facility? | [horse] riding facility
| `type=route`+ `route=horse` | Horseback Riding Route | / | [horse] riding route
| `leisure=track` + `sport=horse_racing` | Horse Racetrack | Horse Racing Track | [horse] raceway, [horse] racecourse

**Comment by Hufkratzer** [comment]
Some change / enhancement requests for the new table:

* `leisure=pitch` + `sport=equestrian`
  * I would swap name and alias (name=Arena, alias=Ring) because
    * this [article ](https://horserookie.com/what-do-you-call-a-place-where-you-ride-horses/) (linked [above](https://github.com/openstreetmap/id-tagging-schema/issues/413#issuecomment-1128356750) by @1ec5 ) correctly says: _"**Arena** This is the most universal term for the location where people ride horses."_
    * according to [Google Trends](https://trends.google.com/trends/explore?date=today%205-y&q=riding%20arena,riding%20ring) the term "riding arena" is searched for on average about twice as often as "riding ring", not only worldwide but also in the US (in most regiones) 
    * [wikipedia](https://en.wikipedia.org/wiki/Equestrian_facility#Schools,_arenas_and_pens) calls them mainly "arenas" (or "pens")
    * the OSM wiki calls them [riding arenas](https://wiki.openstreetmap.org/wiki/Tag:leisure%3Dpitch#Riding_arena), and this cannot be changed to "ring" because British English is used there 
    * all websites which I have listed in https://github.com/openstreetmap/id-tagging-schema/issues/437#issue-1197438977 call them "arenas" and not "ring", see also [here](http://www.inspiredhomeideas.com/horseback-riding-ranch-horse-stables-barns-and-facilities/) (already mentioned above) 
    * in the search results name _Horseback Riding Arena_ will probably be listed before _Horseback Riding Center_ while _Horseback Riding Ring_ would probably be listed after _Horseback Riding Center_ (A before C before R), which may help avoid mistaggings, since mostly riding arenas are mapped
    * it is easier for British translators to translate _Horseback Riding Arena_ to _Horse Riding Arena_ than to translate _Horseback Riding Ring_ to _Horse Riding Arena_
  * possible additional terms: [longeing](https://en.wikipedia.org/wiki/Longeing), jumping/dressage arena/ring
* `leisure=horse_riding`
  * I wouldn't use _Facility_ anymore because it is too general (explained in https://github.com/openstreetmap/id-tagging-schema/issues/437#issue-1197438977)
  * better aliases or search terms may be
    * riding stable / stables
    * horse farm/ranch/park
* `type=route` + `route=horse`
  * possible terms: "horse trail", "trail riding" (compare [mtb](https://github.com/openstreetmap/id-tagging-schema/blob/main/data/presets/type/route/mtb.json))

**Comment by 1ec5** [comment]
I’m not an expert at equestrian sports, so I would be happy to defer to someone who is. The article I linked equates arenas with rings, but the two words are different in other sports, hence my hesitation. There probably isn’t a 100% perfect way to accommodate both experts and laypeople, so either way is fine by me.

The article also distinguishes round pens from arenas, but it’s unclear to me if OSM does. I’ve seen them both mapped as `leisure=pitch` `sport=equestrian` – is that correct?

> in the search results name Horseback Riding Arena will probably be listed before Horseback Riding Center while Horseback Riding Ring would probably be listed after Horseback Riding Center (A before C before R), which may help avoid mistaggings, since mostly riding arenas are mapped

(Maybe in some regions, but there’s a _lot_ of room for improvement in North America. I probably skip over dozens of unmapped round pens every week for lack of time.)

Preset search results are ranked by [edit distance](https://en.wikipedia.org/wiki/Damerau–Levenshtein_distance) (that is, similarity to the search term) and would only be alphabetized in the event of a tie.

**Comment by Hufkratzer** [comment]
> The article I linked equates arenas with rings, but the two words are different in other sports, hence my hesitation.

I admit that this is a problem. But when I looked around the web, I always had the impression that "arena" is used more often than "ring", [example page](https://sportsplanningguide.com/8-best-equestrian-facilities-america/). And I would not make the less used word the main name in iD. (But maybe this was selective perception).

> The article also distinguishes round pens from arenas, but it’s unclear to me if OSM does. I’ve seen them both mapped as `leisure=pitch` `sport=equestrian` – is that correct?

There is a proposal for [leisure=practice_pitch](https://wiki.openstreetmap.org/wiki/Tag:leisure%3Dpractice_pitch) which could be used for [round] pens, but it isn't used for any of them yet (in combination wirh `sport=equestrian)`.

**Comment by tyrasd** [comment]
I've now included the proposed name changes (including the suggestions from https://github.com/openstreetmap/id-tagging-schema/issues/413#issuecomment-1129425366). As far as I can tell, this fixes all the issued listed in the [original post](https://github.com/openstreetmap/id-tagging-schema/issues/413#issue-1160138042), except for the racetrack presets showing up when searching for _horseback_. I'm not 100% sure why exactly these show up (I would assume because they share enough letters to be classified as a partial match), but fixing that would need to be done in iD's code.

**Comment by Hufkratzer** [comment]
Thanks.

https://github.com/openstreetmap/id-tagging-schema/blob/2be67fdc9dffa30e58288af479f62d8f38b001f3/data/presets/leisure/horse_riding.json#L32

should actually be "**riding** stables". It could also be moved to the aliases as "Horseback Riding Stables".

What I do not understand is that for en_GB "Horse Riding Stables" was added as an alias, but the preset is still not listed when searching for "riding stables":

![transifex](https://user-images.githubusercontent.com/5958184/171288088-316009d4-734c-47e8-8561-6a37ba738c4f.png)

![emptyidresult](https://user-images.githubusercontent.com/5958184/171288122-0559d231-71c5-450a-a02e-bab371da1c38.png)

Why is that?

**Comment by 1ec5** [comment]
Support for searching by an alias was added very recently, after the latest v2.20.4 release: openstreetmap/iD@8796a412f8098cde968817e47db928d3cc9d4595. openstreetmap/iD#6139 remains open to track displaying the alias as the name of the matching search result.
